egs/tedlium/s5_r2/local/ted_train_lm.sh

```diff
@@ -74,31 +74,33 @@ if [ $stage -le 0 ]; then
   # note, we can't put it in ${dir}/data/text/, because then pocolm would use
   # it as one of the data sources.
   cut -d " " -f 2-  < data/dev/text  > ${dir}/data/real_dev_set.txt
-  
+
   # get wordlist
   awk '{print $1}' db/cantab-TEDLIUM/cantab-TEDLIUM.dct | sort | uniq > ${dir}/data/wordlist
 fi
 
 order=4
 
-if [ $stage -le 1 ]; then  
-  # decide on the vocabulary.                                                   
-  # Note: you'd use --wordlist if you had a previously determined word-list     
-  # that you wanted to use.                                                     
+if [ $stage -le 1 ]; then
+  # decide on the vocabulary.
+  # Note: you'd use --wordlist if you had a previously determined word-list
+  # that you wanted to use.
   # Note: if you have more than one order, use a certain amount of words as the
-  # vocab and want to restrict max memory for 'sort', 
-  # the following might be a more reasonable setting:                     
-  # train_lm.py --num-word=${num_word} --num-splits=10 --warm-start-ratio=20 ${max_memory} \
-  #             --min-counts='train=2 ted=1' \                               
-  #             --keep-int-data=true --fold-dev-into=ted ${bypass_metaparam_optim_opt} \
-  #             ${dir}/data/text ${order} ${lm_dir} 
+  # vocab and want to restrict max memory for 'sort',
   echo "$0: training the unpruned LM"
-  train_lm.py  --wordlist=${dir}/data/wordlist --num-splits=10 --warm-start-ratio=20  \
+  min_counts='train=2 ted=1'
+  wordlist=${dir}/data/wordlist
+
+  lm_name="`basename ${wordlist}`_${order}"
+  if [ -n "${min_counts}" ]; then
+    lm_name+="_`echo ${min_counts} | tr -s "[:blank:]" "_" | tr "=" "-"`"
+  fi
+  unpruned_lm_dir=${lm_dir}/${lm_name}.pocolm
+  train_lm.py  --wordlist= --num-splits=10 --warm-start-ratio=20  \
```

Comment:
Yes i think so, please modify it like that and see if it runs.
Go at least to the lang/ preparation, in case name changes matter downstream.